src/archivematicaCommon/requirements/base.txt

```diff
@@ -1,2 +1,3 @@
+bagit==1.5.2
```

Comment:
Should we add a MCPClient requirements file?  The dependency is in MCPClient not archivematicaCommon.  Or do we want to let archivematicaCommon manage requirements for MCPClient and have MCPClient require archivematicaCommon?